Eldev

```diff
@@ -10,7 +10,7 @@
 
 (eldev-use-plugin 'autoloads)
 
-(eldev-add-extra-dependencies 'test 'buttercup)
+(eldev-add-extra-dependencies 'test 'buttercup 's)
```

Comment:
Given you've used only a single function from `s`, I'd rather us just add it as some test helper. Less is more. :-)

Reply:
done.